.github/workflows/ci.yml

```diff
@@ -30,21 +30,23 @@ jobs:
     name: JIT compiled tests for Python ${{ matrix.python-version }}
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-python@v5.0.0
-        name: Set up Python ${{ matrix.python-version }}
-
+      - uses: mamba-org/setup-micromamba@v1
+        name: Set up Micromamba environment
```

Comment:
@ikrommyd This is a drive by comment but I would use `mamba-org/setup-micromamba@v1` like you're doing now and then just specify the Python and ROOT version you want like: https://github.com/mamba-org/setup-micromamba?tab=readme-ov-file#create-environment-from-environment-specs

remember to setup the shell defaults though

```
    defaults:
      run:
        # The login shell is necessary for the setup-micromamba setup
        # to work in subsequent jobs.
        # https://github.com/mamba-org/setup-micromamba#about-login-shells
        shell: bash -e -l {0}
```

Reply:
I think I got it. Thanks. I basically want to test against some roofit PDFs so ROOT version shouldn't really matter. I think I can leave it unpinned for now.

Reply:
The only thing I don't like right now is this repetitive use of `shell: micromamba-shell {0}`. I haven't found out if I can just activate the environment and keep it active somehow.

Reply:
@ikrommyd c.f. my comment above in https://github.com/zfit/zfit-physics/pull/69#discussion_r1542178078 that shows how to do this for the entire job.

Reply:
ah I'm stupid. Yeah thanks got it